ms-windows/plugins.nsh

```diff
@@ -8,7 +8,7 @@
 ################################################################################
 
 WriteRegStr HKEY_CURRENT_USER "Software\QGIS\QGIS3\Plugins" "grassplugin" "true"
-WriteRegStr HKEY_CURRENT_USER "Software\QGIS\QGIS3\Plugins" "offlineeditingplugin" "true"
-WriteRegStr HKEY_CURRENT_USER "Software\QGIS\QGIS3\Plugins" "topolplugin" "true"

```

Comment:
@jef-n is this correct?

Reply:
nsis is no longer used (just for 3.16 in osgeo4w v1)

Reply:
ok, I'll remove these changes then. thanks